src/javascript/ellulaitinen-averageOfNumbers.js

```diff
@@ -0,0 +1,15 @@
+// This function calculates the average of numbers
+
+const numbers = [5, 8, -7, 3, -2, 15, 8, 6];
+
+const avgOfNumbers = () => {
```

Comment:
@ellu-laitinen make the function name as same as in the filename, or vice versa.

Reply:
Oops, sorry. Changed it!